system/event-script-engine/src/test/resources/flows/missing-subordinate-flow.yml

```diff
@@ -0,0 +1,39 @@
+flow:
+  id: 'missing-subordinate-flow'
+  description: 'Simplest flow of one task'
```

Comment:
Can you please provide a better writing for description?  I really need a better description to explain the purpose of this test

Reply:
technically it is possible to have infinite nested levels but the time-to-live will stop it from running longer than the configured value.